Core/Core/Helpers/Http.cs

```diff
@@ -128,23 +131,24 @@ public static async Task<bool> Ping(string hostnameOrAddress)
   }
 
   /// <summary>
-  /// Pings and tries gettign data from a specific address to verify it's online. Retries 3 times.
+  ///
   /// </summary>
-  /// <param name="address">The address to use</param>
-  /// <returns>True if the the status code is successful, false otherwise.</returns>
-  public static async Task<bool> HttpPing(string address)
```

Comment:
Quick refactor of this function to throw instead of return true/false, just so we have keep the stack trace + exception message

Reply:
This is was always intended as internal so i'm fine with this.